build.gradle

```diff
@@ -139,6 +153,12 @@ allprojects {
       check('FieldCanBeFinal', net.ltgt.gradle.errorprone.CheckSeverity.OFF)
       check('InsecureCryptoUsage', net.ltgt.gradle.errorprone.CheckSeverity.WARN)
       check('WildcardImport', net.ltgt.gradle.errorprone.CheckSeverity.WARN)
+      check('InlineMeSuggester', net.ltgt.gradle.errorprone.CheckSeverity.OFF)
+
+      // Enabling this check results in large number of parameter changes in ssz submodule. Fix in future PR
+      check('MethodInputParametersMustBeFinal', net.ltgt.gradle.errorprone.CheckSeverity.OFF)
```

Comment:
As it turns out, I don't think there's any actual value in this check. We've never had a problem in Teku because method args weren't final and it wastes a lot of time going back to fix errors it raises.  So I'd just leave it disabled.

Reply:
I've updated the comment that these checks are imported from errorprone-dependency but not required in Teku.